test/testdata/nettemplates/TwoNodes100SecondTestUnupgradedProtocol.json

```diff
@@ -5,12 +5,12 @@
     "Wallets": [
       {
         "Name": "Wallet1",
-        "Stake": 0,
-        "Online": true
+        "Stake": 10,
```

Comment:
This change does not have any side-effects to other tests that use this?

Reply:
No. I checked all the tests that were using this json, and all seems to be happy with this change.